lib/clients/graphql/graphql_client.ts

```diff
@@ -30,7 +31,15 @@ export class GraphqlClient {
       );
     }
 
+    if (params.apiVersion) {
```

Comment:
We don't usually validate this kind of param - we're trusting TS to shout if anything is wrong, but this can be a version string (`"2023-04"`) in pure JS and it will work normally.